src/styles/animation.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
+  max-height: 58px;
+  transition: max-height 0.3s ease-in-out;
+}
+
+.item-exit {
```

Comment:
Consider using more descriptive class names instead of `.item-exit`. For example, `.accordion-item-exit`. 🎨